src/handlers/parseRawDependencyDataHandler.ts

```diff
@@ -84,6 +89,25 @@ function parseTreeNodes(treecontent: string, eol: string, indent: string, prefix
                     }
                     parentNode.addChild(curNode);
                 }
+                const parentFilePath: string = parentNode.uri.path;
+                curFilePath = path.join(parentFilePath, `${curNode.groupId}.${curNode.artifactId}`);
+            }
+            // set uri
+            uri = vscode.Uri.file(curFilePath);
+            uri = uri.with({authority: projectPomPath}); // distinguish dependency in multi-module project
+            if (curNode.omittedStatus.status === "conflict") {
+                curNode.uri = uri.with({query: "hasConflict"});
+                // find all parent and set hasConflict upforward
+                let tmpNode = curNode;
+                while (tmpNode.parent !== undefined) {
+                    const parent = <Dependency> tmpNode.parent;
+                    if (parent.uri.query !== "hasConflict") {
+                        parent.uri = uri.with({query: "hasConflict"});
+                    }
```

Comment:
Pruning: I assume if the parsing is top-down, here you can add `else break;`.  Correct me if I'm wrong.